### DISCUSSION TITLE
Customise XLSX export

### DISCUSSION THREAD

**Discussion by SebBorbely** [inventree/InvenTree#5342] [Q&A]
I am wondering if there is a way to customise the CSV/XLSX export to add or remove columns.
Specifically I am hoping that it is possible to add extra columns to the PO "export order to file" command. Whilst I know that each PO is related to a specific supplier I think it would be very useful if the export also included alternate suppliers and SKUs. This would make it quicker for procurement to compare their options without having to check each part individually. 
Similar to how in a BOM export you get Supplier_0 , SKU_0, Supplier_1, SKU_1 etc..
Removing some columns would also be useful though less important as it is easy to remove columns in excel if not needed.

**Comment by matmair** [comment]
That should not be to difficult but would have to be done in code. We have more flexible input/output options on the radar but are laser-focused on #5212

**Reply by SebBorbely** [reply]
Thanks for the reply.
A, That major UI update looks really promising and I can't wait to use it. I'm really loving Inventree and all the dedication you guys put into it.
B, Just to round off this thread, do you think the example I mentioned above would be covered in your more flexible input/output options and is that an open issue/feature request? If not should I create a feature request so that it doesn't get forgotten and can be developed when there is some time?

**Reply by SchrodingersGat** [reply]
Not sure exactly what might be covered in terms of custom exporting. You would be best served to open a new issue so we can track against it.

**Comment by SergeoLacruz** [comment]
Can't you use the report feature for that. The output is hightly flexible.
